test/specs/login.js

```diff
@@ -1,128 +1,134 @@
 const LoginPage = require('../pageobjects/LoginPage');
+const InventoryPage = require("../pageobjects/ProductsPage");
 const {CREDENTIALS} = require("../helper/credentials");
+const {ImageSource} = require("../helper/inventoryData");
 
-xdescribe('Browser Tests', () => {
+describe('Login Page UI Tests', () => {
 
     it('should have the "Swag Labs" title', async () => {
         await LoginPage.open();
```

Comment:
try to use a beforeAll/beforeEach where you load the login page before running the tests so you don't have to write this line every time